### DISCUSSION TITLE
Add ability to create folders for Queries, WorkBench, Record Set, etc.

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5918] [Feature Requests]
It would be incredibly useful to organize queries and data sets by folder since it would help organization dramatically.

**Requested By:** Corinna P and Dan B at CSIRO

**Comment by maxpatiiuk** [comment]
Related to https://github.com/specify/specify7/issues/2627 (maybe even a duplicate depending on an implementation, but different in scope)
